docs/api/qiskit-ibm-runtime/qiskit_ibm_runtime.options.Options.md

```diff
@@ -40,7 +40,7 @@ Options for the primitives.
 
     Maximum execution time in seconds, which is based on system execution time (not wall clock time). System execution time is the amount of time that the system is dedicated to processing your job. If a job exceeds this time limit, it is forcibly cancelled. Simulator jobs continue to use wall clock time.
 
-    Refer to the [Max execution time documentation](https://docs.quantum-computing.ibm.com/run/max-execution-time#maximum-execution-time). for more information.
```

Comment:
You'll want to revert any changes to the `api` folder because these are generated docs. We'll need to fix the source code itself in qiskit-ibm-runtime.